src/webpack/create-controllers-module.js

```diff
@@ -46,14 +49,25 @@ module.exports = function createControllersModule(config) {
             const controllerMain = packageName + '/' + controllerPackageConfig.main;
             const webpackMode = controllerUserConfig.webpackMode;
 
-            controllerContents +=
-                "\n  '" +
-                controllerReference +
-                '\': import(/* webpackMode: "' +
-                webpackMode +
-                '" */ \'' +
-                controllerMain +
-                "'),";
+            let moduleValueContents = '';
+            if (webpackMode === 'lazy-controller') {
+                hasLazyControllers = true;
+                moduleValueContents = `
+new Promise((resolve, reject) => resolve({ default:
```

Comment:
Perhaps could we put this code in another file, to test it/ease its edition with color?